python/requirements.txt

```diff
@@ -6,6 +6,7 @@
 #
 # setup.py install_requires
 aiohttp==3.7
+aiohttp_cors
```

Comment:
Do we need this? And do you. mind dropping a comment explaining why if we do?

Reply:
We do in that it's already required in `setup.py` people just haven't been following the instructions in setup.py to keep requirements.txt up to date.